### PR TITLE
-unsigned arg: tags uint32/64 with `unsigned

### PR DESCRIPTION
**Pull request by mbacarella**
Closes https://github.com/mransan/ocaml-protoc/issues/185 

~~Still very much working on this. The generation works but haven't instrumented my project to actually run it yet.~~

It's ready to review now.

### CODE REVIEW

**Comment by mbacarella** [comment]
Alright, reworked our my project to actually use this. Works! (and found some eyebrow raising unsigned -> signed coercion issues!)

Anyway, ready for your consideration.

File: src/ocaml-protoc/ocaml_protoc_cmdline.ml [hunk 1]
```diff
@@ -129,6 +129,7 @@ module Cmdline = struct
       (* whether pretty printing is enabled *)
     mutable cmd_line_file_options : File_options.t;
       (* file options override from the cmd line *)
+    unsigned_tag : bool ref;
```
Comment (c-cube) [inline src/ocaml-protoc/ocaml_protoc_cmdline.ml:132]: maybe a comment here would be nice?

File: src/runtime/pbrt.mli [hunk 2]
```diff
@@ -420,9 +437,13 @@ module Pp : sig
   val pp_int32 : formatter -> int32  -> unit
   (** [pp_unit fmt i] formats [i] value *)
 
+  val pp_unsigned_of_int32 : formatter -> [`unsigned of int32]  -> unit
```
Comment (c-cube) [inline src/runtime/pbrt.mli:440]: can you add a `@since 2.4` on both these functions please? This is going to warrant a release.

**Review by c-cube — APPROVED** [review]
Look great. Just a few nitpicks, but this will be merged.

**Comment by mbacarella** [comment]
Thank you @c-cube!

**Comment by mbacarella** [comment]
I'm noticing the failing CI tests. I'm confused, I run `dune build @install` locally and don't get those erorrs.

**Comment by mbacarella** [comment]
Oh, I see, the CI is doing a merge master first.

**Comment by c-cube** [comment]
Thank you!

**Pull request merged**